js/data.js

```diff
@@ -4,7 +4,8 @@ var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
-  nextEntryId: 1
+  nextEntryId: 1,
+  profile: {}
```

Comment:
You should start this with a property for each value that will be stored within it set to `null`, so your data model shows the shape your data will take.